wireapp/wire-ios-mono
# feat: remove developer device details settings selection view model [WPB-6441]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6441" title="WPB-6441" target="_blank"><img alt="Sub-task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10816?size=medium" />WPB-6441</a>  [iOS] Remove `DeveloperDeviceDetailsSettingsSelectionViewModel`
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Solutions

Removed debug settings for E2EI

#### How to Test
1. On a debug build.
2. Shake the device
3. Go to Developer settings
4. Look for Device Details Settings.
5.Expectation - It should not be found

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
